nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/TransformXml.java

```diff
@@ -98,6 +99,16 @@
             .addValidator(StandardValidators.BOOLEAN_VALIDATOR)
             .build();
 
+    public static final PropertyDescriptor SECURE_PROCESSING = new PropertyDescriptor.Builder()
+            .name("secure-processing")
+            .displayName("Secure processing")
+            .description("Whether or not to mitigate various XML-related attacks like XXE (XML External Entity) attacks.")
+            .required(true)
+            .defaultValue("false")
```

Comment:
No worries, I found those comments in the email thread but for some reason they did not show for me in the GitHub UI. I understand the desire for backward compatibility but this is a security fix, so as long as it is well-documented in the processor and the release notes/migration notes for the release that contains this, I think `true` is a safe choice.